x-pack/plugin/sql/src/main/java/org/elasticsearch/xpack/sql/analysis/index/IndexResolver.java

```diff
@@ -470,7 +471,7 @@ public void resolveAsSeparateMappings(String indexWildcard, String javaRegex, bo
             String fieldName = entry.getKey();
             Map<String, FieldCapabilities> types = entry.getValue();
 
-            // ignore size added by the mapper plugin
+            // ignore size added by the mapper plugin or indices with no mapping
```

Comment:
Why "indices with no mapping" would be applicable to this condition? (the `_size` field, when enabled, will be added even if the mapping doesn't have any explicitly defined fields).

Reply:
You're right - the location of the comment is incorrect.